spec/paperclip/storage/s3_spec.rb

```diff
@@ -379,6 +379,54 @@ def counter
     end
   end
 
+  context "An attachment that uses S3 for storage and has styles" do
+    before do
+      rebuild_model(
+        (aws2_add_region).merge({
+          storage: :s3,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.